### DISCUSSION TITLE
fcs/cmdEscFL_1 missing

### DISCUSSION THREAD

**Discussion by memy009** [JSBSim-Team/jsbsim#904] [Q&A]
Hi,

I'm currently trying to run a linearisation for an xml made for a VTOL. When lunching, an error that says 'fcs/cmdEscFL_1 missing' occured. 

My frist question is : What does that parameter means.
My second question is : Is there any reference that explains the meaning of each property of JSBSim cause the reference manual did not explain them well. 

Thank U.

**Comment by seanmcleod** [comment]
`fcs/cmdEscFL_1` sounds like a property created by a particular FCS implementation, i.e. not a standard JSBSim property. Have you looked through the FCS implementation to see where the property is defined and how it is used?

Based on the `Esc` in the name I'm assuming this refers to an Electronic Speed Controller.
